core/variant/array.cpp

```diff
@@ -31,6 +31,7 @@
 #include "array.h"
 
 #include "container_type_validate.h"
+#include "core/math/random_number_generator.h"
```

Comment:
You're not actually using this header here. `Math::rand()` is from `core/math/math_funcs.h`.